gammapy/spectrum/tests/test_spectrum_extraction.py

```diff
@@ -14,21 +14,24 @@
 from ...datasets import gammapy_extra
 from ...image import ExclusionMask
 from ...spectrum import SpectrumExtraction, SpectrumObservation
-
+import logging
+import numpy as np
 
 @pytest.mark.parametrize("pars,results", [
-    (dict(containment_correction=False), dict(n_on=95,
-                                              sigma=19.70,
-                                              aeff=549861.8268659255 * u.m ** 2,
-                                              ethresh=0.4353353353 * u.TeV)),
-    (dict(containment_correction=True), dict(n_on=95,
-                                             sigma=19.70,
-                                             aeff=393356.18322397786 * u.m ** 2,
-                                             ethresh=0.6179 * u.TeV)),
+    (dict(containment_correction=False), dict(n_on=172,
```

Comment:
Does it make sense that you now get more counts (95 -> 172)?
I didn't review this PR, just asking.
If it makes sense to you, fine!